crates/biome_js_analyze/src/services/typed.rs

```diff
@@ -15,12 +18,110 @@ pub struct TypedService {
 }
 
 impl TypedService {
+    pub fn module_0_types(&self) -> &[TypeData] {
+        const EMPTY: &[TypeData] = &[];
+        self.resolver
+            .as_ref()
+            .map_or(EMPTY, |resolver| resolver.modules[0].types())
```

Comment:
Is `[0]` really safe? If so, we can add a `// SAFETY` comment, or use a safer approach to avoid panics

Reply:
It's safe, the module resolver has quite some explainer now on the "module 0" concept. But still a good catch, this method was a leftover from some debugging, so I removed it again :)